stories/core/SideNavigation.stories.tsx

```diff
@@ -82,3 +94,96 @@ export const ActiveItem: Story<SideNavigationProps> = (args) => {
     />
   );
 };
+
+export const Submenu: Story<SideNavigationProps> = (args) => {
```

Comment:
Why "Documents" is not active?
![side-submenu](https://user-images.githubusercontent.com/36186912/137446556-e8b176d5-206e-4116-bd74-7c4d2d1713d5.gif)

Reply:
![image](https://user-images.githubusercontent.com/9084735/137484489-f10eeea5-d559-448f-b278-fa986ee72699.png)